style/testing/avoid_let_spec.rb

```diff
@@ -1,23 +1,27 @@
 # BAD
```

Comment:
Hm, seems like "BAD" would be more consistent with the rest of the guide, but sure.